rwengine/src/engine/GameData.hpp

```diff
@@ -293,6 +293,11 @@ class GameData {
      */
```

Comment:
Add some documentation about why there are 2 sets? And that they are disjoint? (are they?)

Reply:
nit: second animation should be plural as well